baseapp/abci.go

```diff
@@ -457,6 +461,11 @@ func (app *BaseApp) LoadSnapshotChunk(req abci.RequestLoadSnapshotChunk) abci.Re
 
 // OfferSnapshot implements the ABCI interface. It delegates to app.snapshotManager if set.
 func (app *BaseApp) OfferSnapshot(req abci.RequestOfferSnapshot) abci.ResponseOfferSnapshot {
+	if app.snapshotManager == nil {
+		app.logger.Error("snapshot manager not configured")
```

Comment:
ditto

Reply:
...I disagree on this though - this call returns an error result. It seems more correct to log this even as an error?

Reply:
I don't know. When someone will try to call it?  What's the normal flow to call that function?

Reply:
Tendermint will call this when starting a new, empty node with state sync enabled after having found state sync snapshots in the network.

Reply:
So, in that case, we shouldn't log error here. It's not our fault that someone will asks as for a snapshot and we are simply not offering it.

Reply:
So, if the above makes sense, then we should log Info or Warning here.

Reply:
I think you misunderstand. This is called when Tendermint offers a snapshot to the ABCI application to be restored. This is controlled entirely by the local user, who asked Tendermint to run state sync for the local node. This should error, to tell the user why state sync isn't working.